src/apps/content/actions/product-actions.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Dependencies.
+ */
+
+let AppDispatcher = require('../dispatcher/app-dispatcher');
```

Comment:
React's convention is `TitleCase`, the most common node.js one is with `-` and `_` is also an option. I personally don't care much, so lmk what we should continue with.